redhat-developer/app-services-cli
# WIP: feat(generate-config): create config with existing service account

`rhoas generate-config` should not create service-account.

### Verification Steps

1. Generate configuration by specifying `--type` to be `configmap`
```
rhoas generate-config --type=configmap
```
2. It should create a configmap object.
3. Unlike previous version, `rhoas generate-config` shouldn't create a service account.

### ToDo:

- Add validation for flag combinations
- Add validation to check if the service account credentials provided are valid
- Add a message for the breaking change


### Type of change
